vote/turnout
# Fix missing-address handling

I'm not sure why I pulled the call to `get_absentee_contact_info` up the stack; we weren't using it and were just re-calling `get_absentee_contact_info` later inside a try/catch block. This removes the call that was throwing an error for Fayette county (which has no addresses in the LEO database, but does have an email override specified for absentee ballots)

Testing this:
- Make sure you have no addresses for region 433545:

```
SELECT official_address.* FROM official_region
JOIN official_office ON official_office.region_id = official_region.external_id
JOIN official_address ON official_address.office_id = official_office.external_id
WHERE official_region.external_id = 433545;
```

- Make sure you have the Fayette override:

```
INSERT INTO "absentee_absenteeleoemailoverride"("created_at","modified_at","region_id","email")
VALUES
(E'2020-05-17 16:43:21.394057+00',E'2020-05-17 16:43:21.394057+00',433545,E'electionsgroup@fayettecountyga.gov');
```

Without this change, submitting a ballot request in Fayette will fail. With this change, it will succeed.
